assets/css/index.css

```diff
@@ -0,0 +1,41 @@
+.day {
+    display:inline-block;
+    }
+
+.day:hover {
+    background-color: #C1E1C1;
+}
+
+.add-icon {
+    opacity: 50%;
+    height: 40px;
+    width: auto;
+    display: flex;
+}
+
+.day:hover .add-icon {
+    display: flex !important;
+}
+
+.calendar {
+    padding-top: 2.5%;
+    padding-bottom: 2.5%;
+    padding-left: 10%;
+    padding-right: 10%;
+}
+
+.footer-menu {
+    padding-left: 40%;
+    padding-right: 40%;
+    display: flex;
+    flex-flow: row;
+    justify-content: center;
+    justify-content: space-around;
+    font-size: 16px;
+}
+.footer-name {
+    opacity: 50%;
+    display: flex;
+    justify-content: center;
+    font-size: 16px;
+}
```

Comment:
Could you add some comments, please? 
for example:
/* Body Styles */
/* Footer Styles */